docs/rules/prefer-istanbul-ignore-reason.md

```diff
@@ -0,0 +1,27 @@
+# Disallow disabling coverage for whole files (no-istanbul-ignore-file)
+
+This rule raises a warning about "istanbul ignore" comments missing a reason why
+code is ignored.
+
+## Rule Details
+
+The rule finds all comments including patter `istanbul ignore <word>` and raises
```

Comment:
This doesn't read right to me - I assume it's meant to be:

```suggestion
The rule finds all comments matching the pattern `istanbul ignore <word>` and raises
```

(`including` wasn't part of the typo, but I feel `matching` sounds more organic in this sentence)